tbx/project_styleguide/templates/patterns/molecules/pagination/pagination.html

```diff
@@ -0,0 +1,38 @@
+{% if pages.paginator.num_pages > 1 %}
```

Comment:
@engineervix this is hard-coded for now - I am hoping you can fix this in your back-end branch.